knora/dsplib/models/helpers.py

```diff
@@ -410,7 +416,11 @@ def toJsonObj(self) -> dict[str, str]:
 
     def get_externals_used(self) -> dict[str, str]:
         exclude = ["rdf", "rdfs", "owl", "xsd", "knora-api", "salsah-gui"]
-        return {prefix: onto.iri for prefix, onto in self._context.items() if prefix not in exclude}
+        prefixes: dict[str, str] = {}
+        for prefix, onto in self._context.items():
+            if prefix not in exclude:
+                prefixes[prefix] = onto.iri
+        return prefixes
```

Comment:
why remove the elegant dict comprehension? :)

Reply:
Because I find them hard to read ;) I probably tried to understand and rewrote it. But I changed it back now.